taxi/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from taxi.views import index
+
+
+urlpatterns = [
+    path("", index, name="index")
+]
+
+app_name = "taxi"
```

Comment:
nice) but in some files necessary add new line in the end

Reply:
agree, thanks)